### ISSUE TITLE
Relaying Error

### ISSUE THREAD

**Issue by omryozgrappa** [opengsn/gsn#943]
**Describe the bug**

**Steps To Reproduce**
trying to execute "enroll" function on the following contract "0x62b6C0E92917589fA8569841a7544AB79c3b2BD8" on polygon

**Expected behavior**
Expecting the transaction to be relayed through

**Console Logs and/or Screenshots**
```
Relaying errors (1):
https://polygon.3-0-0-beta-3.opengsn.org/v3 => local view call to 'relayCall()' reverted: view call to 'relayCall' reverted in client: {"code":-32603,"message":"Internal JSON-RPC error.","data":{"code":-32000,"message":"err: insufficient funds for gas * price + value: address 0x434EFBCA662f6F846f8DeC3fdE52Fac4c8792e03 have 2550816605859437982 want 3273095523480000000 (supplied gas 12000000)"}} stack:Error: local view call to 'relayCall()' reverted: view call to 'relayCall' reverted in client: {"code":-32603,"message":"Internal JSON-RPC error.","data":{"code":-32000,"message":"err: insufficient funds for gas * price + value: address 0x434EFBCA662f6F846f8DeC3fdE52Fac4c8792e03 have 2550816605859437982 want 3273095523480000000 (supplied gas 12000000)"}}
```

** Link to your project's repository **
In case your repository is open-source and there is already a commit with the code you are having an issue with, it may be helpful to see it to understand what went wrong.

**Additional context**
configuration:
```
{
{logLevel: 'debug'},
paymasterAddress: '0xFcA1200C2f42197539b70d17714eEf98D1af8204',
performDryRunViewRelayCall: false,
minMaxPriorityFeePerGas: 82716215680,
// maxViewableGasLimit: '3157770248985000000',
}
```

**Comment by forshtat** [comment]
Hi @omryozgrappa 
I think this is a known issue in beta.3 where the relay and client require large amounts of ether to be held by the worker and the paymaster to work properly. 
We will try to increase balance of relay server but the real fix will be available in beta.4

**Comment by forshtat** [comment]
This issue must be fixed in `3.0.0-beta.6` by PR: https://github.com/opengsn/gsn/pull/937

Please comment here or open a new issue if you are still experiencing the same issue after an update.
